examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2776,6 +2834,26 @@
         }
       }
     },
+    "examplepbBook": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The resource name of the book.\n\nFormat: `publishers/{publisher}/books/{book}`\n\nExample: `publishers/1257894000000000000/books/my-book`"
+        },
+        "id": {
+          "type": "string",
+          "description": "Output only. The book's ID.",
+          "readOnly": true
+        },
+        "create_time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Output only. Creation time of the book.",
+          "readOnly": true
+        }
+      }
+    },
```

Comment:
What part of this is incorrect?

Reply:
Parameters is currently:

```json
{
    "parameters": [
          {
            "name": "parent",
            "description": "The publisher in which to create the book.\n\nFormat: `publishers/{publisher}`\n\nExample: `publishers/1257894000000000000`",
            "in": "path",
            "required": true,
            "type": "string"
          },
          {
            "name": "body",
            "description": "The book to create.",
            "in": "body",
            "required": true,
            "schema": {
              "$ref": "#/definitions/examplepbBook"
            }
          }
    ],
}
```

Notice that there is no mention of the `book_id`. For clients generated off the swagger.json there is literally no way for them to know they need to provide a `book_id`.

Thus `parameters` should include this:

```json
{
  "name": "book_id",
  "description": "The ID to use for the book.\n\nThis must start with an alphanumeric character",
  "in": "query",
  "required": true,
  "type": "string"
}
```

Note: this is correlated by the reply on my AIP issue here: https://github.com/googleapis/aip/issues/542#issuecomment-656138118